src/Identity.ts

```diff
@@ -47,6 +47,19 @@ export class Identity<A> {
   alt(fx: Identity<A>): Identity<A> {
     return this
   }
+
+  /**
+   * Lazy version of {@link alt}
+   * @since 1.6.0
+   * @param {Lazy<Identity<A>>} fx - thunk
+   * @example
+   * const a = new Identity(1)
+   * assert.deepEqual(a.altL(() => new Identity(2)), a)
+   * @returns {Identity<A>}
+   */
+  altL(fx: Lazy<Identity<A>>): Identity<A> {
```

Comment:
in case of `Identity` is `altL` actually useful?

Reply:
well, I just added it just because `alt` already exists

Reply:
Never mind, please add a test though

Reply:
done